Gemfile

```diff
@@ -9,9 +9,11 @@ gem 'bootstrap', '~> 4.3.1'
 gem 'cancancan', '~> 3.0.1'
 gem 'coffee-rails', '~> 4.2'
 gem 'devise', '~> 4.6', '>= 4.6.2'
+gem 'httparty', '~> 0.17.0'
```

Comment:
Why we need this one?

Reply:
It's used for creating webhook

Reply:
Rails std isn't enough?

Reply:
As for me, httparty's syntax is prettier, and more redeable than Net::HTTP.
So should I rewrite POST request to Trello's API using Net::HTTP?